cEP-0022.md

```diff
@@ -0,0 +1,162 @@
+# Quickstart Green Mode
+
+| Metadata |                                               |
+| -------- | --------------------------------------------- |
+| cEP      | 22                                            |
+| Version  | 1.0                                           |
+| Title    | Quickstart Green Mode                         |
+| Authors  | Ishan Srivastava <mailto:ishan.srt@gmail.com> |
+| Status   | Implementation Due                            |
+| Type     | Feature                                       |
+
+# Abstract
+
+This document describes what is the **Green Mode** for `coala-quickstart` and how
+will this be implemented.
+
+This cEP describes new type of config files which will be generated by
+`coala-quickstart`, how will they be different from the ones currently being
+created, how is this helpful and the action-plan for adding this feature.
+
+# Introduction
+
+Currently `coala-quickstart` generates config files which point out too many
+errors in the code-base of the project. It also asks too many questions
+while generating the config file. The config files are not that specific to
+the project and the errors pointed out by coala using generated config files
+are generally undesirable for the project owners.
+
+Project owners need a configuration file which *agrees* with their codestyle, is
+as specific as possible i.e. can point out maximum errors with least amount
+of manual edits in the future, for future inconsistencies in commits.
+Many organizations don't follow specific code styles like `PEP8`, instead they
+follow their own standards. We need to adapt to those standards with least
```

Comment:
I feel it would be better to split these up into points

Reply:
something with the title like "requirements of green mode"

Reply:
and another section of "Goals", so that we are clear with **why** are we doing this (Goals), and **what** do we want (requirement), and **how** are we going to do this (would be covered in the implementation)